corona-vision/src/utils/api.js

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+
+const API_URL = 'https://dbb7k004t9.execute-api.us-east-1.amazonaws.com/dev/events';
+
+export const postEvents = data => {
+  return axios({
+    method: 'POST',
+    url: API_URL,
+    data
+  });
+};
+
+export const getEvents = query => {
+  return axios({
+    method: 'GET',
+    url: API_URL,
+    query
```

Comment:
does adding the query in the body like this work? I think for most network request libraries (and just RESTful content in general), you want to send the params of a GET request in the URL. This link might be helpful: https://stackoverflow.com/questions/46404051/send-object-with-axios-get-request